ui/index.html

```diff
@@ -28,12 +29,24 @@
       </div>
       <div id="seedContainer" class="container" style="display:none">
          Enter random seed:<br />
-         <input type="text" id="seedTextBox" />
+         <input type="text" id="seedTextBox" maxLength="10" onkeypress="limitKeypress(event,this.value,maxLength)" />
```

Comment:
Some audits use different seeds so maybe we shouldn't limit the `maxLength` here

Reply:
Removed the 'limitKeypress' function to not restrict input length.